common/djangoapps/student/views/management.py

```diff
@@ -690,6 +695,11 @@ def password_change_request_handler(request):
             from openedx.core.djangoapps.user_api.accounts.api import request_password_change
             request_password_change(email, request.is_secure())
             user = user if user.is_authenticated else User.objects.get(email=email)
+            if is_block_user(user):
+                return HttpResponse(
+                    _("This user account has been blocked, please contact administrator."),
```

Comment:
Hi @huynguyen-groove ,
Make the error message consistent such as

'Your account has been disabled.<br/>'
        'If you feel this is a mistake, please contact us at '
        '{support_email}'